language/documentation/examples/experimental/rounding-error/sources/Reserve.move

```diff
@@ -0,0 +1,89 @@
+module NamedAddr::Reserve {
```

Comment:
Comment explaining what this demonstrates.  If it comes from a real contract, which one?  (I don't think it does in this case).